filters/GpsTimeConvert.cpp

```diff
@@ -193,225 +210,174 @@ int GpsTimeConvert::dayStartGpsSeconds(std::tm date)
     durEnd = std::chrono::system_clock::from_time_t(std::mktime(&date));
 
     std::chrono::duration<int> duration;
-    duration = std::chrono::duration_cast<std::chrono::duration<int>>(durEnd
-               - durStart);
+    duration = std::chrono::duration_cast<std::chrono::duration<int>>(durEnd -
+                                                                      durStart);
 
     return duration.count();
 }
 
-void GpsTimeConvert::unwrapWeekSeconds(PointView& view)
+void GpsTimeConvert::unwrapWeekSeconds(PointRef& point)
 {
     // any decrease in time is interpreted as a week rollover
-    for (PointId i = 0; i < (view.size()-1); ++i)
+    while (point.getFieldAs<double>(Dimension::Id::GpsTime) < m_lastTime)
```

Comment:
@hobu, @abellgithub we realized that test is somehow too strict. For some point cloud, you might have little decrease of `GpsTime` (namely when multiple returns).
We introduced another parameter called `wrapped_tolerance` that would update the test like this :
`point.getFieldAs<double>(Dimension::Id::GpsTime) + m_wrappedTolerance < m_lastTime`

Is that ok if I put this modification in this PR ?

Reply:
> Is that ok if I put this modification in this PR ?

👍